index.html

```diff
@@ -0,0 +1,220 @@
+              <li class="list-items">JavaScript</li>
+            </ul>
+            <a class="Project-link" href="#"
+              >See this project<i
+                class="fa fa-long-arrow-right fa-2x"
+                aria-hidden="true"
+              ></i>
+            </a>
+          </div>
+        </div>
+        <div class="info">
+          <img src="img/macbook-pro-img2.jpg" alt="" />
+          <h2>Project name goes here</h2>
+          <div>
+            <ul class="list">
+              <li class="list-items">HTML/CSS</li>
+              <li class="list-items">Ruby on Rails</li>
+              <li class="list-items">JavaScript</li>
+            </ul>
+            <a class="Project-link" href="#"
+              >See this project<i
+                class="fa fa-long-arrow-right fa-2x"
+                aria-hidden="true"
+              ></i>
+            </a>
+          </div>
+        </div>
+      </section>
+      <section class="About">
+        <h2>About</h2>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+          aliquip ea eiusmod
+        </p>
+        <button type="submit">
+          Get my CV <i class="fa fa-download" aria-hidden="true"></i>
+        </button>
+      </section>
+      <div class="Code-lang">
+        <h3>Languages</h3>
+        <ul>
+          <li>JavaScript(ES5/ES6)</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+      </div>
+      <div class="Code-lang2">
+        <h3>Languages</h3>
+        <ul>
+          <li>JavaScript(ES5/ES6)</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+      </div>
+      <div class="Code-lang3">
+        <h3>Languages</h3>
+        <ul>
+          <li>JavaScript(ES5/ES6)</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+      </div>
+      <section class="Info-form">
+        <img src="img/image_geometry_1.png" alt="" />
+        <div class="footh2">
+          <h2>Get started</h2>
+        </div>
+        <div class="footP">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna.
+        </div>
+        <form action="https://formspree.io/f/{form_id}" method="post">
+          <label class="name" for="name">Your name</label>
+          <input
+            class="form"
+            type="text"
+            name="name"
+            placeholder=""
+            maxlength="30"
+            required
+          />
+          <label class="email" for="email">Your email</label>
+          <input
+            class="form"
+            type="email"
+            name="email"
+            placeholder=""
+            maxlength="30"
+            required
+          />
```

Comment:
- [ ] Additionally, let's try as much as possible to stick to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=67%3A553&t=IBsN4pmH4ABH3raE-0). For this case we need to ensure that the label is displayed as intended. Kindly try your best to match the Figma design.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/222087563-decbc0c9-634c-48e7-a691-605c943fb5e5.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/222087638-2926564a-8004-4d8d-8a0f-90268cd57136.png)

Reply:
- [ ] As requested by the previous reviewer, please stick to the figma. Check the image below showing where the differences are.
![form](https://user-images.githubusercontent.com/99656687/222104545-00f28165-8313-4fbc-b464-b13439bde796.png)